marklogic-data-hub-central/ui/src/pages/noMatchRedirect.tsx

```diff
@@ -1,28 +1,88 @@
-import React, {useContext, useEffect} from "react";
-import {Result} from "antd";
+import React, {useContext, useEffect, useState} from "react";
 import {withRouter} from "react-router-dom";
 import {UserContext} from "../util/user-context";
 import {HCButton} from "@components/common";
+import styles from "./noMatchRedirect.module.scss";
+import oopsIcon from "../assets/oopsIcon.png";
+import {Row, Col} from "react-bootstrap";
+import {ArrowLeftShort} from "react-bootstrap-icons";
 
 const NoMatchRedirect = ({history}) => {
 
   const {user, clearErrorMessage} = useContext(UserContext);
+  const [copyClicked, setCopyClicked] = useState("Copy");
+  const [errorBodyText, setErrorBodyText] = useState("");
 
   useEffect(() => {
     clearErrorMessage();
+    setErrorBodyText(user.error.message);
   }, []);
 
-  const backToHomePage = () => {
+  const goBack = () => {
     return user.authenticated ? history.push("/tiles") : history.push("/");
   };
+
+  const copyToClipboard = () => {
+    setCopyClicked("Copied!");
+    navigator.clipboard.writeText(errorBodyText);
+    setTimeout(function () { setCopyClicked("Copy"); }, 3000);
+  };
+
   return (
-    <Result
-      status={404}
-      title="404"
-      subTitle="Sorry, the page you visited does not exist."
-      extra={<HCButton variant="primary" aria-label="back home" onClick={backToHomePage}>Back Home</HCButton>}
-    />
+    <div className="container-fluid">
+      <Row>
+        <Col className="col-md-3" onClick={goBack}>
+          <div className={`d-flex align-items-center cursor-pointer`} style={{float: "right"}}>
```

Comment:
Instead of use float: right; it's better approach using justify-content: end; in a flex container.

Reply:
bootstrap class: justify-content-end